ui.frontend/src/customFunctions.js

```diff
@@ -42,7 +42,52 @@ function externalize(url) {
     }
 }
 
+/**
+ * Validates if the given URL is correct
+ * @param url
+ * @returns {boolean}
+ */
+function validateURL(url) {
+    try {
+        const validatedUrl = new URL(url, window.location.href);
+        return (validatedUrl.protocol === 'http:' || validatedUrl.protocol === 'https:');
```

Comment:
there is an issue with the code that Ravi is fixing 

@ravise5

Reply:
IE11 is anyways not supported for core components right ?